server/serverAPI/controller/InventoryController.ts

```diff
@@ -52,6 +51,60 @@ export default class InventoryController extends BaseUserController {
         return ingredientSchema;
     }
 
+    protected returnByExpirationDate(inventory: IInventoryIngredient[], isReverse: boolean): any {
+        let itemsWithExpirationDate: IInventoryIngredient[] = [];
+        let itemsWithoutExpirationDate: IInventoryIngredient[] = [];
+        
+        inventory.forEach(item => {
+            if (item.expirationDate) {
+                itemsWithExpirationDate.push(item);
+            }
+            else
+                itemsWithoutExpirationDate.push(item);
+        });
+
+        itemsWithExpirationDate.sort((a, b) => b.expirationDate! - a.expirationDate!);
```

Comment:
Can you add a comment saying whether this is sorted as expiring sooner to expiring later or vice versa? ty